diff -uNr zlib-1.2.13/CMakeLists.txt zlib-1.2.13_patched/CMakeLists.txt
--- zlib-1.2.13/CMakeLists.txt	2023-05-25 11:28:38.408919456 +0200
+++ zlib-1.2.13_patched/CMakeLists.txt	2023-05-25 11:28:48.916933298 +0200
@@ -6,8 +6,8 @@
 set(VERSION "1.2.13")
 
 set(INSTALL_BIN_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE PATH "Installation directory for executables")
-set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib" CACHE PATH "Installation directory for libraries")
-set(INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include" CACHE PATH "Installation directory for headers")
+set(INSTALL_LIB_DIR "dev/lib" CACHE PATH "Installation directory for libraries")
+set(INSTALL_INC_DIR "dev/include" CACHE PATH "Installation directory for headers")
 #set(INSTALL_MAN_DIR "${CMAKE_INSTALL_PREFIX}/share/man" CACHE PATH "Installation directory for manual pages")
 #set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
 
